Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -333,7 +337,7 @@ class CustomBuildTaskTerminal implements Pseudoterminal {
     async open(_initialDimensions: TerminalDimensions | undefined): Promise<void> {
         telemetry.logLanguageServerEvent("cppBuildTaskStarted");
         // At this point we can start using the terminal.
-        this.writeEmitter.fire(`Starting build...${this.endOfLine}`);
+        this.writeEmitter.fire(localize("starting_build", "Starting build...{0}", this.endOfLine));
```

Comment:
Maybe add the endOfLine outside of the call to localized?  Otherwise, translators are going to ask us what the {0} is for, as they might wonder if it's relevant to the translation.